fendermint/app/config/default.toml

```
@@ -63,7 +63,7 @@ port = 8545
 # TODO: fetch this value from genisis if possible.
 subnet_id = "/r0"
 
-[ipc.config]
+[ipc.topdown]
 # The number of blocks behind chain head to propose parent finality.
 chain_head_delay = 10
```

```suggestion
chain_head_delay = 900
```
Let's be brutal here and assume the worst.